batch2/deployment.yaml

```diff
@@ -39,8 +39,10 @@ spec:
            value: /deploy-config/deploy-config.json
          - name: BATCH_WORKER_IMAGE
            value: "{{ batch2_worker_image.image }}"
-         - name: BATCH_NAMESPACE
+         - name: HAIL_DEFAULT_NAMESPACE
            value: "{{ default_ns.name }}"
+         - name: HAIL_BATCH_PODS_NAMESPACE
```

Comment:
These also have to be added below. There's two deployment configs here and I think you only got one of them.

Reply:
Fixed, thanks!  Might be more issues...